### DISCUSSION TITLE
Is there any issue with morph target?

### DISCUSSION THREAD

**Discussion by atul161** [google/filament#6479] [Q&A]
Hi team,

Thank you for your constant support. I think there is some major issue with the morph target. I am getting a dark spot all over the body when setting up morph targets in avatar(glb). I am getting this error in every second phone.

I am calling set morph weights from the render method. Before that, I am also enabling morphing in builder.

Is there something I am missing out?

Thankyou!

**Comment by bejado** [comment]
Which device(s) and are you able to upload the glb file you're using?

**Reply by ritik619** [reply]
@bejado 
It is happening on almost all android devices We've tried on Samsungs and Oppo ,One plus and realme devices
[63aef9c3d16b67196c5966c2 (1).glb.zip](https://github.com/google/filament/files/10487968/63aef9c3d16b67196c5966c2.1.glb.zip)

**Reply by ritik619** [reply]
Also we were not able to replicate this issue on Android Emulator
It's a intermittent issue 
Sometime it loads properly sometime it doesn't

**Reply by ritik619** [reply]
@bejado  any idea on this?
